src/riak_kv_w1c_sup.erl

```diff
@@ -0,0 +1,55 @@
+%% -------------------------------------------------------------------
+%% Copyright (c) 2015 Basho Technologies, Inc. All Rights Reserved.
+%%
+%% This file is provided to you under the Apache License,
+%% Version 2.0 (the "License"); you may not use this file
+%% except in compliance with the License.  You may obtain
+%% a copy of the License at
+%%
+%%   http://www.apache.org/licenses/LICENSE-2.0
+%%
+%% Unless required by applicable law or agreed to in writing,
+%% software distributed under the License is distributed on an
+%% "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+%% KIND, either express or implied.  See the License for the
+%% specific language governing permissions and limitations
+%% under the License.
+%%
+%% -------------------------------------------------------------------
+-module(riak_kv_w1c_sup).
+
+-behaviour(supervisor).
+
+-export([start_link/0]).
+-export([init/1]).
+
+%%%===================================================================
+%%% API functions
+%%%===================================================================
+
+-spec(start_link() ->
+    {ok, Pid :: pid()} | ignore | {error, Reason :: term()}).
+start_link() ->
+    supervisor:start_link({local, ?MODULE}, ?MODULE, []).
+
+%%%===================================================================
+%%% Supervisor callbacks
+%%%===================================================================
+
+-spec(init(Args :: term()) ->
+    {ok, {SupFlags :: {RestartStrategy :: supervisor:strategy(),
+        MaxR :: non_neg_integer(), MaxT :: non_neg_integer()},
+        [ChildSpec :: supervisor:child_spec()]
+    }}).
+init([]) ->
+    Workers = riak_kv_w1c_worker:workers(),
+    Children = [
+        {
+            Name, {riak_kv_w1c_worker, start_link, [Name]},
+            permanent, 5000, worker, [riak_kv_w1c_worker]
+        } || Name <- tuple_to_list(Workers)],
```

Comment:
Sure, I figured that out in the other code. I think we need to punt on this nit but make a todo for the number of workers to be configurable.